libs/designsystem/src/lib/components/radio/radio.component.scss

```diff
@@ -13,6 +14,24 @@ $default-checkbox-radio-size: map.get(utils.$checkbox-radio-sizes, 'md');
   @return ($target-height - $radio-icon-size) * 0.5;
 }
 
+// TODO: Move this to a shared location, e.g. in libs/core/src/scss/interaction-state directory
+@function _get-hover-color($variant, $loudness: interaction-state.$default-loudness, $flip: false) {
+  $_background: utils.get-color($variant, $getValueOnly: true);
+
+  $_lightness: 0% !default;
+
+  @if $flip {
+    $_lightness: utils.get-loudness($loudness);
+  } @else {
+    // negative number means "make it darker"
+    $_lightness: -1 * utils.get-loudness($loudness);
+  }
+
+  $hover-color: #{color.scale($_background, $lightness: $_lightness)};
+
+  @return $hover-color;
+}
+
```

Comment:
See https://github.com/kirbydesign/designsystem/pull/2122/files#r834288241

Reply:
Do it!